{"text": "2024-03-16 06:23:41 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.048341", "seconds": 1.048341}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:23:41.720603+08:00", "timestamp": 1710541421.720603}}}
{"text": "2024-03-16 06:23:51 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:10.797804", "seconds": 10.797804}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:23:51.470066+08:00", "timestamp": 1710541431.470066}}}
{"text": "2024-03-16 06:23:53 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:13.298402", "seconds": 13.298402}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:23:53.970664+08:00", "timestamp": 1710541433.970664}}}
{"text": "2024-03-16 06:23:56 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xdf in position 2: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xdf in position 2: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:00:15.672061", "seconds": 15.672061}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xdf in position 2: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xdf in position 2: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:23:56.344323+08:00", "timestamp": 1710541436.344323}}}
{"text": "2024-03-16 06:23:58 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:17.685618", "seconds": 17.685618}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:23:58.357880+08:00", "timestamp": 1710541438.35788}}}
{"text": "2024-03-16 06:24:27 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:47.229124", "seconds": 47.229124}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:24:27.901386+08:00", "timestamp": 1710541467.901386}}}
{"text": "2024-03-16 06:24:27 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:47.229835", "seconds": 47.229835}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:24:27.902097+08:00", "timestamp": 1710541467.902097}}}
{"text": "2024-03-16 06:24:58 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:18.320275", "seconds": 78.320275}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:24:58.992537+08:00", "timestamp": 1710541498.992537}}}
{"text": "2024-03-16 06:25:00 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:20.284036", "seconds": 80.284036}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:25:00.956298+08:00", "timestamp": 1710541500.956298}}}
{"text": "2024-03-16 06:25:03 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xe3 in position 4: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xe3 in position 4: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:01:22.364920", "seconds": 82.36492}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xe3 in position 4: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xe3 in position 4: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:25:03.037182+08:00", "timestamp": 1710541503.037182}}}
{"text": "2024-03-16 06:25:05 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:24.624522", "seconds": 84.624522}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:25:05.296784+08:00", "timestamp": 1710541505.296784}}}
{"text": "2024-03-16 06:25:34 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:54.168237", "seconds": 114.168237}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:25:34.840499+08:00", "timestamp": 1710541534.840499}}}
{"text": "2024-03-16 06:25:34 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:54.168796", "seconds": 114.168796}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:25:34.841058+08:00", "timestamp": 1710541534.841058}}}
{"text": "2024-03-16 06:26:05 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:25.252994", "seconds": 145.252994}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:05.925256+08:00", "timestamp": 1710541565.925256}}}
{"text": "2024-03-16 06:26:08 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:27.378255", "seconds": 147.378255}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:08.050517+08:00", "timestamp": 1710541568.050517}}}
{"text": "2024-03-16 06:26:10 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xc7 in position 1: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xc7 in position 1: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:02:29.735614", "seconds": 149.735614}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xc7 in position 1: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xc7 in position 1: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:10.407876+08:00", "timestamp": 1710541570.407876}}}
{"text": "2024-03-16 06:26:12 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:31.917427", "seconds": 151.917427}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:12.589689+08:00", "timestamp": 1710541572.589689}}}
{"text": "2024-03-16 06:26:42 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:03:01.459764", "seconds": 181.459764}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:42.132026+08:00", "timestamp": 1710541602.132026}}}
{"text": "2024-03-16 06:26:42 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:03:01.460544", "seconds": 181.460544}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:26:42.132806+08:00", "timestamp": 1710541602.132806}}}
{"text": "2024-03-16 06:27:13 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:32.557162", "seconds": 212.557162}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:27:13.229424+08:00", "timestamp": 1710541633.229424}}}
{"text": "2024-03-16 06:27:13 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3842), thread 'MainThread' (127777978588032): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic_free_2/epic_free_2/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:32.693856", "seconds": 212.693856}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_free_2/epic_free_2/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3842), thread 'MainThread' (127777978588032):", "module": "claim", "name": "__main__", "process": {"id": 3842, "name": "MainProcess"}, "thread": {"id": 127777978588032, "name": "MainThread"}, "time": {"repr": "2024-03-16 06:27:13.366118+08:00", "timestamp": 1710541633.366118}}}
